client/shipping-label/views/steps/rates/index.js

```diff
@@ -26,11 +110,11 @@ const RatesStep = ( { form, values, available, storeOptions, labelActions, error
 
 RatesStep.propTypes = {
 	form: PropTypes.object.isRequired,
-	values: PropTypes.array.isRequired,
-	available: PropTypes.array.isRequired,
+	values: PropTypes.object.isRequired,
+	available: PropTypes.object.isRequired,
 	labelActions: PropTypes.object.isRequired,
 	storeOptions: PropTypes.object.isRequired,
-	errors: PropTypes.array.isRequired,
+	errors: PropTypes.object.isRequired,
```

Comment:
Added in https://github.com/Automattic/woocommerce-connect-client/pull/533/commits/8a55916c833e85b9d4969775df397441ff772645.